lesson11/main.js

```diff
@@ -17,16 +17,20 @@ const loaded = () => {
 
 window.onload = () => {
   loading();
```

Comment:
IMO

I thought `window.onload` was unnecessary.
Currently, I think why don't you put `loading()` in the try catch clause?

Reply:
Thank you for the advice too! I realized what this code do with you guys's advice.
I delited  `window.onload` and move `loading` in the try part.
If I made more mistake, please kindly tell me.
[bad0128](https://github.com/yuka830/js-lessons/pull/13/commits/bad01286f0b6029ebf65693a32af81ea6e08a43e)
[CodeSandbox](https://codesandbox.io/s/lesson11-d7w3u?file=/main.js:0-1541)